ports/stm32/main.c

```diff
@@ -528,6 +529,10 @@ void stm32_main(uint32_t reset_mode) {
     machine_i2s_init0();
     #endif
 
+    #if defined(STM32WL)
+    subghz_init();
+    #endif
```

Comment:
IMO this should go next to rfcore_init().  

Then there should be a separate subghz_deinit() which just clears the callback handler, before gc_sweep_all() is run.  Otherwise a hard interrupt that comes in after gc_sweep_all() is called, will call bad data.

Reply:
> IMO this should go next to rfcore_init().
>
> Then there should be a separate subghz_deinit() which just clears the callback handler, before gc_sweep_all() is run. Otherwise a hard interrupt that comes in after gc_sweep_all() is called, will call bad data.

Oh, I missed that root pointers aren't reset automatically! Will definitely fix this.

I think it is also desirable to hard reset the radio peripheral on soft reset, similar to the other peripherals - as it might be mid-transmit or receive otherwise. Will do this the deinit function as well, though.